pkg/datapath/linux/requirements.go

```diff
@@ -183,16 +183,6 @@ func CheckMinRequirements() {
 				log.Warn("llc version was compiled in debug mode, expect higher latency!")
 			}
 		}
-		// /usr/include/gnu/stubs-32.h is installed by 'glibc-devel.i686' in fedora
```

Comment:
As far as I know，this section is for people that are not running cilium in a container; 
I have a question about whether need to delete this section code?

Reply:
After this PR, Cilium does not depend on glibc header at all, so yes, that code can be removed.